services/brig/test/integration/API/Internal.hs

```diff
@@ -217,8 +217,8 @@ testFeatureConferenceCallingByAccount (Opt.optSettings -> settings) mgr db brig
           . (^?! Aeson.key "conferenceCalling")
           . responseJsonUnsafe @Aeson.Value
 
-  check $ ApiFt.WithStatusNoLock ApiFt.FeatureStatusEnabled ApiFt.ConferenceCallingConfig
-  check $ ApiFt.WithStatusNoLock ApiFt.FeatureStatusDisabled ApiFt.ConferenceCallingConfig
+  check $ ApiFt.WithStatusNoLock ApiFt.FeatureStatusEnabled ApiFt.ConferenceCallingConfig ApiFt.FeatureTTLUnlimited
+  check $ ApiFt.WithStatusNoLock ApiFt.FeatureStatusDisabled ApiFt.ConferenceCallingConfig ApiFt.FeatureTTLUnlimited
```

Comment:
@fisx on top of the integration tests that check TTL expiration results in the flag being reverted back to default? 🤔 

Also since we changed the record used for the requests, we can indeed use it via PUT, so I've started simplifying that on the receiving end of stern, so we no longer need to be passing the TTL around as an extra parameter 👍

Reply:
> @fisx on top of the integration tests that check TTL expiration results in the flag being reverted back to default? thinking

no, if you have a test for that i'm happy, must have missed that.